harcownik-frontend/src/components/userRequests/UserEfficiency.jsx

```diff
@@ -55,8 +56,8 @@ function UserEfficiency() {
     setBadges(badges);
   };
 
-  function getBadgeNameById(id) {
-    const badge = badges.find((b) => b.id === id);
+  function getBadgeNameById(Id) {
+    const badge = badges.find((b) => b.id === Id);
```

Comment:
variable with name b is kind of useless, would be better have more common with context names